sdcm/sct_events.py

```diff
@@ -462,6 +462,35 @@ def __str__(self):
             super(SpotTerminationEvent, self).__str__(), self)
 
 
+class PrometheusAlertEvent(SctEvent):  # pylint: disable=too-many-instance-attributes
+    def __init__(self, event_type:str, annotations: dict = None, startsAt=None, endsAt=None, updatedAt=None, fingerprint: str = None,
+                 status: dict = None, generatorURL: str=None, labels: dict = None, severity=Severity.NORMAL, **kwargs):
```

Comment:
- I don't think we need the generatorURL in the Event, it doesn't have any valuable information
- if we have alert, the severity should be WARNING
- remove `**kwargs` (if there is something that needed, it should be passed in the constractor

Reply:
fingerprint - one alert rule can issue multiple alert instances with same name and starts_at, in this case fingnerprint is good thing to track down specific alert instance

Reply:
Other items are fixed